### DISCUSSION TITLE
Can combobox filtering use "contains" as well as "starts with"?

### DISCUSSION THREAD

**Discussion by leetheman** [microsoft/fluentui-blazor#746] [Q&A]
Hi there, 

This is almost a perfect replacement for my current blazor component library.  Just one thing:

The combobox currently filters using "startswith".  Is there any option to filter using "contains"?

Cheers

Lee

**Comment by dvoituron** [comment]
Hi and thank you for using our components :-)

The next release will contain a **FluentAutocomplete** component with this possibility (and more).
See https://preview.fluentui-blazor.net/Autocomplete

**Reply by Hwiet** [reply]
Can FluentAutocomplete allow selection of a single item without showing it as a badge? I'm looking for something that looks like the _Interactive with debounce example_ in the [FluentSearch documentation](https://www.fluentui-blazor.net/Search) that shows the suggestions in a hovering listbox like FluentAutocomplete and FluentCombobox.

![image](https://github.com/user-attachments/assets/d684e9af-6797-4398-b054-e692ae50544a)

**Reply by vnbaaij** [reply]
No, AutoComplete always uses the badge for displaying selected item(s)

**Reply by dvoituron** [reply]
Correct.

But you can customize the selected items to not display the "remove button".

```razor
@* Template used with each Selected items *@
<SelectedOptionTemplate>
    @context.LastName
</SelectedOptionTemplate>
```
